cas-server-support-openid/src/main/java/org/jasig/cas/support/openid/authentication/principal/OpenIdServiceResponseBuilder.java

```diff
@@ -137,10 +137,10 @@ protected Response buildAuthenticationResponse(final ServerManager serverManager
 
         final Message response = serverManager.authResponse(this.parameterList, id, id,
                 successFullAuthentication, true);
+        //this will always include an association handle, either generated in
```

Comment:
I might ask that you turn these into proper javadocs so it's easier to understand from the outside.